extend german normalizer

emphasize opened the pull request:
## Problem:
With the usage of whisper (most of germans were using google up until now i assume) we get a lot of hyphenated "word-combinations" like `"15-Minuten-Timer"`.

This is an unexpected behaviour and a problem in the intent stage that can't be dealt with subsequently.

will be brought to `classifiers` after review

emphasize committed:
extend german normalizer

JarbasAl commented on lingua_franca/lang/parse_de.py:
```diff
@@ -1156,6 +1156,14 @@ class GermanNormalizer(Normalizer):
     with open(resolve_resource_file("text/de-de/normalize.json")) as f:
         _default_config = json.load(f)
 
+    def remove_symbols(self, utterance):
```
this method should go into the base normalizer and get a flag for all langs in the .json

the more performant way would be
```
text = "15-minute-timer"
translating = str.maketrans('', '', string.punctuation)
new_string = text.translate(translating)
```

emphasize replied:
~~`str.maketrans(''.join(symbols), len(symbols)*" ")`~~
gotcha (altough i have doubts with `_`/`=`)

Will add this tomorrow

emphasize committed:
add symbols

emphasize committed:
add symbols

emphasize committed:
fix nice date utils (#60)

* fix nice date utils

* remove doubled import

* added tests

JarbasAl committed:
Increment Version

emphasize committed:
move to quebra tokenizer (#61)

JarbasAl committed:
Increment Version

emphasize committed:
Merge branch 'dev' into add/extended_ger_normalizing

emphasize committed:
replace regex

emphasize commented:
Rebased, adjusted to the latest developments and flagged every langs `remove_symbols: true` (`normalize.json`; where possible)
Changed the default in the Normalizer class to `True`
https://github.com/OpenVoiceOS/ovos-lingua-franca/blob/f16bb4dc6edd38da93df415ca2361c33623fa904/lingua_franca/lang/parse_common.py#L52-L54

replaced (german hyphen splitting) regex pattern to be less error prone

Should be ready to go
